localstack/cli/localstack.py

```diff
@@ -180,12 +180,19 @@ def cmd_stop():
 )
 @publish_invocation
 def cmd_logs(follow: bool):
+    from localstack.utils.bootstrap import LocalstackContainer
     from localstack.utils.docker_utils import DOCKER_CLIENT
 
     container_name = config.MAIN_CONTAINER_NAME
+    logfile = LocalstackContainer(container_name).logfile
 
     if not DOCKER_CLIENT.is_container_running(container_name):
         console.print("localstack container not running")
+        console.print("printing logs from previous run")
+        if os.path.exists(logfile):
```

Comment:
maybe we could move this into the check so we don't print the message that we are printing logs when there are no logs to print :-)

```suggestion
        if os.path.exists(logfile):
            console.print("printing logs from previous run")
```